hydra-cluster/test/Test/Hydra/Cluster/FaucetSpec.hs

```diff
@@ -12,44 +12,50 @@ import Hydra.Cluster.Faucet (returnFundsToFaucet, seedFromFaucet, seedFromFaucet
 import Hydra.Cluster.Fixture (Actor (..))
 import Hydra.Cluster.Scenarios (EndToEndLog (..))
 import Hydra.Cluster.Util (keysFor)
-import Hydra.Ledger.Cardano (genVerificationKey)
+import Hydra.Ledger.Cardano (genKeyPair, genVerificationKey)
 import Hydra.Logging (showLogsOnFailure)
 import Test.QuickCheck (elements, generate)
 
 spec :: Spec
 spec = do
-  describe "seedFromFaucet" $
```

Comment:
Should: keep the `describe` contexts to not need to mention `seedFromFaucet` on each test case

this will also make thee diff here smaller as "seedFromFaucet and returnFundsToFaucet work together" was unchanged.